proto/src/determined/api/v1/model.proto

```diff
@@ -72,6 +72,8 @@ message GetModelsRequest {
   // Limit the models to those made by the users with the following usernames.
   repeated string users = 9;
 
+  // Limit models in the following worksapce.
+  optional string workspace_name = 12;
```

Comment:
we should change lookups to use `workspace_id` (or support both), see thread: https://determined-ai.slack.com/archives/C03LE5Z8R46/p1672864845972189?thread_ts=1672772755.830529&cid=C03LE5Z8R46

Reply:
will allow for both workspace id and workspace name in model registry.  Because workspace name would be easier for CLI users. Workspace id will be the default.